components/alert/js-alerts/js-alerts.css

```diff
@@ -0,0 +1,142 @@
+@import url(/css/base.css);
+@import url(/css/utility.css);
+
+.et_js-alerts-container{
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    margin: 4rem 0rem;
+}
+
+/* js-dismissible alerts start here */
+.dismissible-container{
+    align-items: center;
+    background-color: var(--white-color);
+    border: 1px solid var(--light-gray);
+    display: flex;
+    flex-direction: column;
+    height: fit-content;
+    margin: 2rem 0rem;
+    padding: 0rem 0rem 2rem 0rem;
+    width: 90%;
+}
+
+.dismissible-container h3{
+    color: var(--secondary-color);
+    font-size: 1.2rem;
+    text-align: center;
+}
+
+.alert{
+    border: 1px solid var(--light-gray);
+    color: var(--light-gray);
+    display: flex;
+    justify-content: space-between;
+    margin: 1rem 0rem;
+    padding: 1rem;
+    width: 90%;
+}
+
+.et_dismissible-simple-primary-alert{
+    background-color: var(--primary-color);
+}
+
+.et_dismissible-simple-secondary-alert{
+    background-color: var(--lightest-blue);
+}
+
+.et_dismissible-success-secondary-alert{
+    background-color: var(--lightest-green);
+}
+
+.et_dismissible-danger-secondary-alert{
+    background-color: var(--lightest-red);
+}
+
+.et_dismissible-warning-secondary-alert{
+    background-color: var(--lightest-yellow);
+}
+
+.et_dismissible-info-secondary-alert{
+    background-color: var(--lightest-turquiose);
+}
+
+.fa-times-circle:hover{
+    color: var(--black);
+}
+/* js-dismissible alerts end here */
+
+/* js-dismissible alerts with strong color starts here */
+.strong-bg>*{
+    color: var(--white-color);
+}
+
+.et_dismissible-strong-primary-alert{
+    background-color: var(--black);
+}
+
+.et_dismissible-strong-secondary-alert{
+    background-color: var(--secondary-color);
+}
+.et_dismissible-strong-success-alert{
+    background-color: var(--green);
+}
+.et_dismissible-strong-danger-alert{
+    background-color: var(--light-red);
+}
+.et_dismissible-strong-warning-alert{
+    background-color: rgb(248, 211, 45);
+}
+.et_dismissible-strong-info-alert{
+    background-color: var(--turquoise);
+}
+.strong-dismiss:hover{
+    color: var(--lightest-gray);
+}
+/* js-dismissible alerts with strong color ends here */
+
+/* js-advanced-dismissible alerts start here */
+.advanced-alert{
+    border: 1px solid var(--light-gray);
+    color: var(--light-gray);
+    display: grid;
+    grid-template-columns: 10% 80% 10%;
+    margin: 1rem 0rem;
+    width: 90%;
+}
+.et_advanced-dismissible-primary-alert{
+    background-color: var(--black);
+}
+.alert-icon{
+    align-items: center;
+    background: rgba(245, 242, 242, 0.1);
+    display: flex;
+    justify-content: center;
+}
+.alert-message{
```

Comment:
Follow only one type of naming

Reply:
okay will look into it!! and will make the required changes, Thankyou so much for the review!!